### PR TITLE
Update documentation to reflect the latest CLI functionality

### PR DESCRIPTION
**Pull request by fsat**
- Built-in resolvers: URI and Bintray
- Bundle shorthand expression
- Orchestration of bundle deployments using CLI

### CODE REVIEW

File: src/main/play-doc/operation/CLI.md [hunk 1]
```diff
@@ -90,52 +90,6 @@ shazar ./visualizer-poll-interval.sh
 
 The configuration is now ready to be loaded along with a bundle. Configuration is always provided as the second param to the `conduct load` command.
 
-## Deploying bundles
```
Comment (huntc) [inline src/main/play-doc/operation/CLI.md:93]: I'm wondering if this section should be retained, with a separate section on orchestrating bundle deployment. WDYT? Have you been through the flow reading from the CLI section, to the next section etc?

Comment (fsat) [inline src/main/play-doc/operation/CLI.md:93]: I was thinking whether _Deploying bundles_ section should stay here or separate section myself.

What made me pull it out into a separate section is I want to consolidate all deployment related topics into one deployment section. Also the deployment section being a top level link called _Deploying Bundles_ under Operations is a plus, I think.

I did read to see how the flow feels like reading from CLI to the next section. I feel it's less jarring this way compared to having _Deploying bundles_ in a separate section from shorthand and orchestration.

I initially started having everything under CLI section, and it does feel that it becomes too long read.

Perhaps maybe we can split the section:
- _CLI_ section explains how to install the CLI tool and what it is for
- _Deploying Bundles_ section explains how to deploy
- _Customising Deployment Configuration_ section explains how to override config (i.e. shazar command)
- _Listing Bundle Endpoints_ section explains about `conduct services`

**Comment by huntc** [comment]
@fsat some of this for 1.0?

**Comment by fsat** [comment]
Made available to 1.0 via #147

**Pull request merged**